### PR TITLE
Adding pseudonetcdf recipe

### PR DESCRIPTION
**Pull request by barronh**
Adding a conda-recipe for PseudoNetCDF. PseudoNetCDF is a pure python package that provides access to many scientific data formats through a NetCDF common data model.

### CODE REVIEW

**Comment by barronh** [comment]
Please let me know how to proceed.

**Comment by barronh** [comment]
This is my first conda-forge recipe, so I am not sure what the protocol is for getting this approved now that it passes all tests.

Please advise, thanks.

File: recipes/pseudonetcdf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (jhamman) [inline recipes/pseudonetcdf/meta.yaml]: @barronh - I think you can safely remove the commented out lines at this point.

**Comment by jhamman** [comment]
@barronh - in the past, I've pinged `@conda-forge/staged-recipes` when I have a completed recipe.

**Comment by barronh** [comment]
@conda-forge/staged-recipes - this PR is ready for review. Thanks.

**Comment by barronh** [comment]
@conda-forge-coordinator - I'm not sure how to get this recipe in the feedstock. Please advise.

File: recipes/pseudonetcdf/meta.yaml (hunk 1 above)
Comment (barronh) [reply recipes/pseudonetcdf/meta.yaml]: Thanks, done.

**Comment by barronh** [comment]
@synapticarbors - I saw you were instrumental at getting several python recipes merged. Can you help me understand what I need to do to get this one merged? I was hoping for a recipe to support integration of this package into xarray.

Thanks,

xarray PR: https://github.com/pydata/xarray/pull/1905

File: recipes/pseudonetcdf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  run:
+    - python
+    - numpy
+    - pandas
+    - netcdf4
+    - scipy
+    - pyyaml
+    - matplotlib
+
+test:
```
Comment (synapticarbors) [inline recipes/pseudonetcdf/meta.yaml]: This recipe looks good to me except for one outstanding question. I see that there is a command line interface to this library with a bunch of tools. Should those be tested?

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

I feel like stuff in `scripts` have perhaps not played well with `noarch: python` due to mangling of `#!/usr/bin/env bash`, but that may be an upstream conda issue. I'll leave it up to you if you want to try to include those tests or if you're fine with just testing the library functionality. After that I'm happy to merge.

Comment (barronh) [reply recipes/pseudonetcdf/meta.yaml]: @synapticarbors - Thanks! In many ways, the scripts are legacy convenience routines. In the next version, many will be removed from the package distribution and added to a separate repo for users. I'd rather not hold up the recipe to test them.

For what its worth, I've had good experience with the scripts on linux and mac. On windows, the performance of the scripts has been more mixed and version dependent. It is really this cross-platform complication that has led me to redesign -- making the scripts less of a central usecase.

Comment (synapticarbors) [reply recipes/pseudonetcdf/meta.yaml]: Sounds reasonable. I'm going to go ahead and merge as is.

**Comment by synapticarbors** [comment]
@barronh -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Comment by barronh** [comment]
Thanks, I've looked at the docs, but I'll review again.  I'm sure some details will make more sense when I am in process.

**Pull request merged**